tests/Aspire.Dashboard.Tests/TelemetryRepositoryTests/ApplicationKeyTests.cs

```diff
@@ -10,7 +10,7 @@ public class ApplicationKeyTests
 {
     [Theory]
     [InlineData("name", "instanceid", "instanceid", null)]
-    [InlineData("name-abc", "name-abc", "name", "abc")]
+    [InlineData("name-abc", "name-abc", "name-abc", null)]
```

Comment:
Looks good like an improvement to me.

(for context: matching names between telemetry and resources and has been a struggle since day 1 of Aspire. Make sure there are tests and playground apps for your scenarios to help avoid future changes regressing your scenario)